tests/unit/test_cesm_driver.py

```diff
@@ -1,4 +1,5 @@
 import os
+from sys import platform
```

Comment:
We have one in `vic/drivers/cesm/bld/`, this will likely change but we probably don't need to add anything else at this point.